src/node.cpp

```diff
@@ -325,6 +332,26 @@ Node* Node::unserialize(MegaClient* client, string* d, node_vector* dp)
 
     ptr = n->attrs.unserialize(ptr);
 
+    PublicLink *plink = NULL;
+    if (version_flag >= 1)
+    {
```

Comment:
But in the future, if the version value changes, let's say `version_flag=2` for an additional data, it will mean that the node is exported, and it may not.
I think is better to have one specific byte for that, and to use the reserved byte exclusively for the version number.